<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>social_media</class>
 <widget class="QDialog" name="social_media">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="social_media_title">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Social Media Points</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="event_name_label">
      <property name="text">
       <string>Post Title:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="name_sheet_label">
      <property name="text">
       <string>Name Sheet:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="name_sheet_upload">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="point_sheet_label">
      <property name="text">
       <string>Point Sheet:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="point_sheet_upload">
        <property name="text">
         <string>File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="points_label">
      <property name="text">
       <string>Points per Post:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>351</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
  </widget>
  <widget class="QLabel" name="backLabel">
   <property name="geometry">
    <rect>
     <x>-320</x>
     <y>-310</y>
     <width>1331</width>
     <height>1021</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>backLabel</zorder>
  <zorder>social_media_title</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>verticalLayoutWidget_5</zorder>
  <zorder>verticalLayoutWidget_7</zorder>
  <zorder>next_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
